packages/flutter/lib/src/material/input.dart

```diff
@@ -304,16 +308,26 @@ class _InputContainerState extends State<InputContainer> {
     );
     EdgeInsets margin = new EdgeInsets.only(bottom: bottomHeight - (bottomPadding + bottomBorder));
 
-    stackChildren.add(new AnimatedContainer(
-      margin: margin,
-      padding: padding,
-      duration: _kTransitionDuration,
-      curve: _kTransitionCurve,
-      decoration: new BoxDecoration(
-        border: border,
-      ),
-      child: config.child,
-    ));
+    Widget divider;
+    if (config.hideDivider) {
+      divider = new Container(
+        margin: margin,
+        padding: padding,
```

Comment:
I think these two will be slightly different heights.  Is that what you intended?